server/src/routes/root.js

```diff
@@ -3,8 +3,10 @@ const router = express.Router();
 
 const example = require("./example.route");
 const authRoutes = require("./auth.routes");
+const signinRoutes = require("./auth.signin");
 
 router.use("/example", example);
 router.use("/signup", authRoutes);
+router.use("/signin", signinRoutes);
```

Comment:
As I said after putting the authSignin to the auth.route.js you can just remove the sign in thing and the rest will work on it's own as it will import all of auth routes